src/Jab.Unity/Jab.Unity.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <IsPackable>false</IsPackable>
```

Comment:
Is this correct? How will users get this dll then?

Reply:
Unity users have only 2 options: downloading/building and dragging manually the dll into their Unity project or using an npm package (which I created [here](https://github.com/AlonTalmi/Jab.Unity))

Unity doesn't support Nuget unfortunately

Reply:
If you can build the Jab.Unity.dll and Jab.Attributes.dll into any directory in this repository then we only need to add a package.json like [this](https://github.com/AlonTalmi/Jab.Unity/blob/main/package.json) inside that directory and then users can install it using:
https://github.com/pakrym/jab?path={the path to that directory}

Reply:
Ohhh and we need to add Unity specific meta file for each dll by opening them inside Unity

Reply:
Is the meta file generation a one time operation or is it required for every new dll build?

Reply:
One time